src/wms/src/main/java/org/geoserver/wms/featureinfo/GetFeatureInfoKvpReader.java

```diff
@@ -114,13 +114,20 @@ public Object read(Object req, Map kvp, Map rawKvp) throws Exception {
                     + "It should be a proper subset of those instead");
```

Comment:
Copyright headers were not updated, they should state "2014 - 2015" after this change